test/api/surface_test.js

```diff
@@ -485,7 +485,7 @@ describe('Echo metadata', function() {
     call.end();
   });
   it('shows the correct user-agent string', function(done) {
-    var version = require('grpc/package.json').version;
+    var version = require('../any_grpc')['$implementationInfo'].server.corePjson.version;
```

Comment:
I'm pretty sure the user agent should match the client version, not the server version.